pkg/agent/controller/traceflow/traceflow_controller_test.go

```diff
@@ -625,7 +625,8 @@ func TestStartTraceflow(t *testing.T) {
 			tf: &crdv1alpha1.Traceflow{
 				ObjectMeta: metav1.ObjectMeta{Name: "tf3", UID: "uid3"},
 			},
-			expectedErrLog: "Traceflow tf3 has neither source nor destination Pod specified",
```

Comment:
I think this parameter can be removed now given the only two cases using it are changed, and there doesn't seem a reason why we should check error log in the future.